src/exodus/wallet.cpp

```diff
@@ -86,6 +86,11 @@ SigmaSpend Wallet::CreateSigmaSpend(PropertyId property, SigmaDenomination denom
     return SigmaSpend(SigmaMintId(*mint, DefaultSigmaParams), mint->chainState.group, anonimitySet.size(), proof);
 }
 
+bool Wallet::EraseSigmaMint(const SigmaMintId &id)
+{
+    return CWalletDB(walletFile).EraseExodusMint(id);
```

Comment:
We should throw instead if it fail so it will more easy to use from the caller site.